test/deposit_cancellation.test.ts

```diff
@@ -0,0 +1,329 @@
+import {
+  A_DAI,
+  STKAAVE_WHALE,
+  INCENTIVES_CONTROLLER,
+  LENDING_POOL,
+  DAI_WHALE,
+  EMISSION_MANAGER,
+} from "../constants/addresses";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import chai, { expect } from "chai";
+import { Contract, ContractFactory, providers, BigNumber } from "ethers";
+import hre, { starknet, network, ethers } from "hardhat";
+import {
+  StarknetContractFactory,
+  StarknetContract,
+  HardhatUserConfig,
+  Account,
+  StringMap,
+} from "hardhat/types";
+import { solidity } from "ethereum-waffle";
+import config from "../hardhat.config";
+
+import { TIMEOUT } from "./constants";
+
+chai.use(solidity);
+
+const MAX_UINT256 = hre.ethers.constants.MaxInt256;
+
+// Amount of dai and usdc to transfer to the user. Issued twice for aDai and aUsdc
+const DAI_UNIT = 1000n * BigInt(10 ** 18);
+const daiAmount = 300n * DAI_UNIT;
+
+describe("AToken deposit cancellation", async function () {
+  this.timeout(TIMEOUT);
+
+  const networkUrl =
+    (config as HardhatUserConfig).networks?.l1_testnet?.url ||
+    "http://localhost:8545";
```

Comment:
this default value is already defined in `hardhat.config.ts`. is it mandatory to define it here as well?